opensubtitles/common.go

```diff
@@ -0,0 +1,143 @@
+package opensubtitles
+
+import "time"
+
+type Subtitles struct {
```

Comment:
Codacy found an issue: [exported type Subtitles should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)